lib/models/notices.js

```diff
@@ -54,6 +54,14 @@ function processCBNotice(ncr) {
             }
             n[attr.name] = attr.value;
             n[attr.name + '__type'] = attr.type;
+
+            if (attr.metadata) {
+                Object.keys(attr.metadata).forEach(function(md) {
+                    if (attr.metadata[md].value) {
+                        n[attr.name + '__metadata__' + md] = attr.metadata[md].value;
```

Comment:
If the value of the metadata "item" is an object it works the same than if it were a number or a string ... It is "saved" under the `attr__metadata__name` key